### ISSUE TITLE
java.lang.NoClassDefFoundError      code -1

### ISSUE THREAD

**Issue by peterodavies27atgmailcom** [MultiMC/MultiMC5#3346]
System Information
-----------------------------
MultiMC version:
0.6.11-1430
Operating System:
windows 10
Summary of the issue or suggestion:
----------------------------------------------
MC crashes

What should happen:
------------------------------
MC shouldn't crash

Steps to reproduce the issue (Add more if needed):
-------------------------------------------------------------
1.Launch modpack

2.Wait

3.

Suspected cause:
---------------------------
Something is wrong w/ rftoolspower

Logs/Screenshots:
----------------------------
[//]: # (Please refer to https://github.com/MultiMC/MultiMC5/wiki/Log-Upload for instructions on how to attach your logs.)
https://pastebin.com/gfzGNEjE
https://snipboard.io/xvHChF.jpg
Additional Info:
none
---------------------------

**Comment by QuImUfu** [comment]
rftoolspower is version 1.1.2, should be 1.2.0

**Comment by peterodavies27atgmailcom** [comment]
oh ok thanks

**Comment by peterodavies27atgmailcom** [comment]
[21:25:59] [Client thread/WARN] [enderio]: TConstruct, you fail again, muhaha! The world is mine, mine!

log: https://pastebin.com/FcDctZp8

**Comment by QuImUfu** [comment]
materialis is trying to register an "blinding" trait that is already registered from some other mod. This is kind of hard to debug, as many mods you include may do such thing.  
If you are willing to upload that pack somewhere, i would find out which mod for you by searching thru all mods using something like `for f in ./*; do echo "$f :\n"; unzip -p $f | strings | grep blinding; done;` under bash. You can do that yourself using wsl or using an equivalent powershell script, instead.

**Comment by peterodavies27atgmailcom** [comment]
https://easyupload.io/gspxt1

**Comment by peterodavies27atgmailcom** [comment]
it's a zip because I can only do like 30 files

**Comment by QuImUfu** [comment]
soo... this took a while. I had to disable SimpleTcon because it conflicts with materialis. i had to build modified versions of Tinker's Construct and Tinker's Aether to fix bugs, disable lots of conflicting materials in different configs (i am not 100% sure i got all). I had to add JustEnoughIDs because it ran out of blockids. Then it finally gets to the point where it tries to load models and crashes with Out of memory errors, because i have not enough memory. I hope you have at least ~8GB memory free, otherwise you'll need to remove some mods anyway.  
I'm not sure if this will run, but its closer for sure :) https://dl.dropbox.com/s/i19wpxh49ypref7/Tinkers%202-fixed%3F.zip
